manifests/params.pp

```diff
@@ -64,4 +64,6 @@
 
   $qpid_url = "amqp:ssl:${::fqdn}:5671"
   $candlepin_event_queue = 'katello_event_queue'
+
+  $is_parent = true
```

Comment:
trying to update for this change https://github.com/Katello/puppet-pulp/pull/108